board-of-directors.tex

```diff
@@ -28,21 +28,21 @@ \section{Number, Election, and Tenure}
 
 At the regular annual meeting, the existing Directors shall elect Directors for
 terms commencing on the first day of the next August and continuing for three
-years and until their respective successors are elected and take office.
-Persons who are Directors by virtue of election are referred to as `elected
-Directors' in these Bylaws.  The number of elected Directors shall not be less
-than four or more than ten.  In addition, unless already Directors by virtue of
-election, the President, the Treasurer, and the House Manager of the Eta Tau
-Chapter of Beta Theta Pi Fraternity shall be ex officio Directors of the
-Corporation during the times such persons occupy their offices as President,
-Treasurer, and House Manager, respectively, of the Eta Tau Chapter of Beta Theta
-Pi Fraternity.  These three Directors are referred to as `ex officio Directors'
-in these Bylaws.
+years and until their respective successors are elected and take office. Persons
+who are Directors by virtue of election are referred to as `elected Directors'
+in these Bylaws. The number of elected Directors shall not be less than four or
+more than ten. In addition, unless already Directors by virtue of election, the
+President, the Treasurer, and the House Manager of the Eta Tau Chapter of Beta
+Theta Pi shall be ex officio Directors of the Corporation during the times such
+persons occupy their offices as President, Treasurer, and House Manager,
+respectively, of the Eta Tau Chapter of Beta Theta Pi. These three Directors are
+referred to as `ex officio Directors' in these Bylaws.
 
 In order to be elected and in order to hold office, each elected Director must
 be a person who was conferred a Bachelor's Degree or Master's Degree by
-Worcester Polytechnic Institute, and at the time so conferred was an initiated
-Brother in good standing of the Eta Tau Chapter of Beta Theta Pi Fraternity.
+Worcester Polytechnic Institute, and at the time so conferred was either an
```

Comment:
I wonder if we shouldn't simply limit it to those in good standing with the chapter. I assume the default standing of anyone is "good", and through Kai, or just a censure for those not in the chapter, one would go to not in good standing

Reply:
Your assumption sounds correct to me. I deliberately worded this so it depends on you having been in good standing at the time that you graduated, not currently. That prevents the Chapter from being able to manipulate who is eligible by declaring alumni to be not in good standing. They would have to produce evidence that you weren't in good standing when you graduated. A good example of _not_ in good standing would be if someone was on suspension at the time that he graduated. That would permanently bar him from being a Director.